### ISSUE TITLE
FormViewController always in edit mode

### ISSUE THREAD

**Issue by OskarGroth** [xmartlabs/Eureka#1410]
When populating a FormViewController with MultivaluedSections, it is set to edit mode by default. From the Demo project:

![skarmavbild 2018-02-07 kl 15 44 45](https://user-images.githubusercontent.com/1417019/35922438-e0f505ca-0c1d-11e8-81fa-12d5881643b7.png)

Trying to set it to normal state in the Demo:

>  class MultivaluedController: FormViewController {
>         override func viewDidLoad() {
>          super.viewDidLoad()
>          self.setEditing(false, animated: false)
>          self.tableView.setEditing(false, animated: false)

Does not work. No difference. However, calling from main queue:

> class MultivaluedController: FormViewController {
>     override func viewDidLoad() {
>         super.viewDidLoad()
>         DispatchQueue.main.async {
>             self.setEditing(false, animated: false)
>             self.tableView.setEditing(false, animated: false)
>         }

Solves it:

![skarmavbild 2018-02-07 kl 15 46 17](https://user-images.githubusercontent.com/1417019/35922527-1bfb3fb8-0c1e-11e8-8d08-5b2a34e5f843.png)

## TL;DR
**Issue:** 
FormViewController always in edit mode. Has to be unset from main thread.
**Expected:**
FormViewController should not be in edit mode by default. Should be able to set/unset without calling from main.

**Comment by mtnbarreto** [comment]
you should do:

```swift
override func viewWillAppear(_ animated: Bool) {
      super.viewWillAppear(animated)
      self.tableView.setEditing(false, animated: false)
}
```
